revscoring/utilities/test.py

```diff
@@ -0,0 +1,116 @@
+"""
+``revscoring test -h``
+::
+
+    Tests a scorer model.  This utility expects to get a file of
+    tab-separated feature values and labels from which to test a model.
+
+    Usage:
+        train_test -h | --help
+        train_test <scorer_model> <features> [-p=<kv>]... [-s=<kv>]...
+                   [--version=<vers>]
+                   [--values-labels=<path>]
+                   [--model-file=<path>]
+                   [--label-type=<type>]
+                   [--scale]
+                   [--debug]
+
+    Options:
+        -h --help               Prints this documentation
+        <scorer_model>          Classpath to an the MLScorerModel to construct
+                                and train
+        <features>              Classpath to an list of features to use when
+                                constructing the model
+        -p --parameter=<kv>     A key-value argument pair to use when
+                                constructing the scorer_model.
+        -s --statistic=<kv>     A test statistic argument to use to evaluate
+                                the scorer model against the test set.
+        --version=<vers>        A version to associate with the model
+        --values-labels=<path>  Path to a file containing feature values and
+                                labels [default: <stdin>]
+        --model-file=<math>     Path to write a model file to
+                                [default: <stdout>]
+        --label-type=<type>     Interprets the labels as the appropriate type
+                                (int, float, str, bool) [default: str]
+        --scale                  Features should be scaled to a common range
+        --debug                 Print debug logging.
+"""
+import json
+import logging
+import sys
+
+import docopt
+import yamlconf
+
+from . import util
+from ..scorer_models.statistics import TestStatistic
+
+logger = logging.getLogger(__name__)
+
+
+def main(argv=None):
+    args = docopt.docopt(__doc__, argv=argv)
+
+    logging.basicConfig(
+        level=logging.INFO if not args['--debug'] else logging.DEBUG,
+        format='%(asctime)s %(levelname)s:%(name)s -- %(message)s'
+    )
+
+    ScorerModel = yamlconf.import_module(args['<scorer_model>'])
+    features = yamlconf.import_module(args['<features>'])
+
+    version = args['--version']
+
+    model_kwargs = {}
+    for parameter in args['--parameter']:
+        key, value = parameter.split("=")
+        model_kwargs[key] = json.loads(value)
+
+    test_statistics = []
+    for stat_str in args['--statistic']:
+        test_statistics.append(TestStatistic.from_stat_str(stat_str))
+
+    scorer_model = ScorerModel(
```

Comment:
c.f. the [MLScorerModel.load](https://github.com/wiki-ai/revscoring/blob/master/revscoring/utilities/score.py#L32) line from the `score` utility for loading a model from a file.